h/util/markdown.py

```diff
@@ -102,7 +102,7 @@ def sanitize(text):
     return cleaner.clean(text)
 
 
-def _linkify_target_blank(attrs, new=False):
+def _linkify_target_blank(attrs, new=False):  # pylint: disable=unused-argument
```

Comment:
Why this one? Is it passed by key word?

Reply:
Yeah passed in as a keyword arg and I don't think we should change the name of the argument (would we change it back if it was used again?)